### PR TITLE
feat: added metrics to roles build, bond, contribute and govern

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Added metrics to various role pages:
* Build
* Bond
* Contribute
* Govern

## Screenshots/Recordings

* Build
image

* Bond
image

* Contribute
image

* Govern
image

### CODE REVIEW

File: components/ContributePage/ContributeMetrics.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          />
+          Total Olas Contributors
+        </div>
+        <ExternalLink
+          className="font-extrabold text-6xl"
+          href="https://operate.olas.network/contracts"
+          target="_blank"
+          hideArrow
+        >
+          320
```
[the remark this answers is not in the document]
Comment (atepem) [reply components/ContributePage/ContributeMetrics.jsx:26]: Yes, we unfortunately can't make this number dynamic unitl [we can track DAAs in Contribute](https://trello.com/c/iCRpLOvG)

File: components/GovernPage/GovernMetrics.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        ? veOLASCirculatingSupply.value
+        : null,
+    veOLASHolders:
+      veOLASHolders.status === 'fulfilled' ? veOLASHolders.value : null,
+  };
+};
+
+export const GovernMetrics = () => {
+  const { data: metrics } = usePersistentSWR(
+    'bondActivityMetrics',
```
Comment (Tanya-atatakai) [inline components/GovernPage/GovernMetrics.jsx:32]:
```suggestion
    'governActivityMetrics',
```

File: components/GovernPage/GovernMetrics.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    <SectionWrapper customClasses="mt-16">
+      <Card className="flex flxe-col p-2 mx-auto border border-purple-200 rounded-full text-xl max-w-3xl rounded-2xl bg-gradient-to-t from-[#F1DBFF] to-[#FDFAFF]">
+        {data.map((item, index) => {
+          const { id, imageSrc, labelText, value, source } = item;
+          let borderClassName = '';
+          if (index == 0)
+            borderClassName +=
+              'max-sm:border-b-1.5 md:border-r-1.5 border-purple-200';
+
+          const getValue = () => {
```
[the remark this answers is not in the document]
Comment (Tanya-atatakai) [reply components/GovernPage/GovernMetrics.jsx:66]: I think we need to make it all reusable somehow, we copy-pasted metrics here and there so many times already, just hope we won't change the design soon

**Review by Tanya-atatakai — APPROVED** [review]
tiny comment, lgtm

**Review by DavidMinarsch — APPROVED** [review]

File: components/GovernPage/GovernMetrics.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (atepem) [reply components/GovernPage/GovernMetrics.jsx:66]: I've tried to refactor the code to have the metrics card reusable but the small differences in the designs make it a bit hard to do... Can probably attempt it tomorrow though

**Pull request merged**